tests/framework/context/test_context.py

```diff
@@ -486,11 +485,6 @@ def test_validate_layers_error(layers, conflicting_datasets, mocker):
             {"a": {"a.c": {"a.c.b": 4}}},
             {"a": {"a.a": 1, "a.b": 2, "a.c": {"a.c.a": 3, "a.c.b": 4}}},
         ),
-        (
-            {"a": OmegaConf.create({"b": 1}), "x": 3},
-            {"a": {"c": 2}},
-            {"a": {"b": 1, "c": 2}, "x": 3},
-        ),
```

Comment:
I remove this test since it's not meaningful as we don't expect any OmegaConf object. I keep the test that checking no OmegaConf is in the params.